src/trace/implementations/ord.rs

```diff
@@ -163,11 +161,13 @@ impl<T: Columnation> RetainFrom<T> for TimelyStack<T> {
 
 /// An immutable collection of update tuples, from a contiguous interval of logical times.
 #[derive(Abomonation)]
-pub struct OrdValBatch<L: Layout> {
+pub struct OrdValBatch<L: Layout, C> {
```

Comment:
I'm reading this, and .. while I can probably figure it out I have no clue what `C` is meant to be. Would you be willing to add some doc text about it? It's a bit awkward that it plays no role for some time, leaving no hints about what it is.

Reply:
I've read a bit more, and .. it seems like this is mostly a newtype hack? Like, `C` could be `bool` and `String` and line up with the `Vec` and `TimelyStack` merge batchers, if the `impl`s said that. It feels a bit weird!

Reply:
I think we could make the merge batcher part of the layout and it would resolve all these issues. But it also feels odd because it's not really part of the layout... What I can't wrap my head around is how to get from an update container on the layout to a merge batcher implementation.

Reply:
Nope, I can't see how to make it part of the layout. We don't know the `Batch` yet, and the batcher needs to be parametrized with `Self` a.k.a. the batch type :/